build.gradle

```diff
@@ -177,18 +177,19 @@ sonarqube {
 	properties {
 		property "sonar.projectName", "rd-user-profile-sync-api"
 		property "sonar.projectKey", "RDPROFILESYNCAPI"
-		property "sonar.jacoco.reportPath", "${project.buildDir}/jacoco/test.exec"
-		property "sonar.jacoco.itReportPath", "${project.buildDir}/jacoco/integration.exec"
+		property "sonar.coverage.jacoco.xmlReportPaths", "${project.buildDir}/reports/jacoco/test/jacocoTestReport.xml"
 		property "sonar.pitest.mode", "reuseReport"
 		property "sonar.pitest.reportsDirectory", "build/reports/pitest"
 		property "sonar.exclusions", "src/main/java/uk/gov/hmcts/reform/profilesync/config/**," +
 				"src/main/java/uk/gov/hmcts/reform/profilesync/client/**," +
-			    "src/main/java/uk/gov/hmcts/reform/profilesync/ProfileSyncApplication.java,"
-
+			    "src/main/java/uk/gov/hmcts/reform/profilesync/ProfileSyncApplication.java," +
+				"src/main/java/uk/gov/hmcts/reform/profilesync/service/impl/ProfileUpdateServiceImpl.java" +
+				"src/main/java/uk/gov/hmcts/reform/profilesync/util/UserProfileSyncJobScheduler.java"
```

Comment:
Sonarcloud thinks I am adding the code changes in these classes but I am not, so to temporarily get the build working I am going to exclude them. They will be covered in QA concerns for Profile Sync next sprint.